nbconvert/filters/filter_links.py

```diff
@@ -0,0 +1,26 @@
+#!/usr/bin/env python3
+"""A pandoc filter used in converting notebooks to Latex.
+Converts links between notebooks to Latex cross-references.
+"""
+import re
+
+from pandocfilters import RawInline, applyJSONFilters
+
+def resolve_references(source):
+    return applyJSONFilters([resolve_one_reference], source)
+
+def resolve_one_reference(key, val, fmt, meta):
+    """
+    """
```

Comment:
Let's either put a docstring in or remove the empty docstring (I'm happy for this to have no docstring).